u_parrots.go

```diff
@@ -567,22 +585,44 @@ func (uconn *UConn) generateRandomizedSpec(WithALPN bool) (ClientHelloSpec, erro
 		p.Extensions = append(p.Extensions, &alpn)
 	}
 
-	if tossBiasedCoin(0.66) {
+	if tossBiasedCoin(0.62) || p.TLSVersMax == VersionTLS13 {
+		// always include for TLS 1.3, since TLS 1.3 ClientHellos are often over 256 bytes
+		// and that's when padding is required to work around buggy middleboxes
```

Comment:
But might we generate a client hello less than 256 bytes before padding? Then we might stick out?

Reply:
padding is a conditional extension that doesn't get added, if client hello size is not in the F5-terminator-failure-range

Reply:
So this block says: with 62% chance, add padding iff client hello size is in the F5-failure range? Or is it just saying "with 62% add padding"

Reply:
> with 62% chance, add padding iff client hello size is in the F5-failure range

yes. if it's tls 1.2

if it's tls 1.3:
with 100% chance, add padding iff client hello size is in the F5-failure range